src/main/java/github/grace000/erversay/response/ResponseSender.java

```diff
@@ -0,0 +1,18 @@
+package github.grace000.erversay.response;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.io.PrintWriter;
+
+public class ResponseSender {
+
+    public void send(OutputStream outputStream, String response) throws IOException {
+        PrintWriter output = new PrintWriter(new OutputStreamWriter(outputStream, "UTF-8"), true);
+
+        output.println(response);
+        output.flush();
+    }
+
+
+}
```

Comment:
Lots of unneeded extra spacing happening in here. You might wanna look into a linter? For pointing stuff like this out in the future. Not a requirement but could be helpful.